pkg/deploy/dev-workspace-config/dev_workspace_config_test.go

```diff
@@ -1328,3 +1409,194 @@ func TestReconcileDevWorkspaceConfigDeploymentStrategy(t *testing.T) {
 		})
 	}
 }
+
+func TestReconcileDevWorkspaceProjectCloneCOnfig(t *testing.T) {
+	const testNamespace = "eclipse-che"
+	testMemLimit := resource.MustParse("2Gi")
+	testCpuLimit := resource.MustParse("1000m")
+	testMemRequest := resource.MustParse("1Gi")
+	testCpuRequest := resource.MustParse("500m")
+
+	type testCase struct {
+		name                       string
+		cheProjectCloneConfig      *chev2.Container
+		expectedDevWorkspaceConfig *controllerv1alpha1.ProjectCloneConfig
+		existingDevWorkspaceConfig *controllerv1alpha1.ProjectCloneConfig
```

Comment:
This is a nitpick and very subjective, but it might be worth renaming these two fields to something like `expectedDWProjectCloneConfig` and `existingDWProjectCloneConfig`.

I got a bit confused later on in the test when you did a [diff check](https://github.com/eclipse-che/che-operator/pull/1687/commits/e7159d0d30f915a14c354b979c65133f5a367cd8#diff-698b095064585b533923485dccc65654810ebbf08649fc81eb10ec4c99a7cc77R1597):

`diff := cmp.Diff(testCase.expectedDevWorkspaceConfig, dwoc.Config.Workspace.ProjectCloneConfig) `

At a glance, it looks like we're comparing the DWOC and the project clone config.